ui/app/templates/components/pki/config-pki.hbs

```diff
@@ -16,10 +16,23 @@
 
 <MessageError @model={{this.config}} @errors={{this.errors}} />
 <form {{action "save" this.section on="submit"}} class="box is-shadowless is-marginless is-fullwidth has-slim-padding">
-  {{#each (get this.config (concat this.section "Attrs")) as |attr|}}
-    <FormField @attr={{attr}} @model={{this.config}} @data-test-field={{attr.name}} />
-  {{/each}}
-  <div class="field is-grouped box is-fullwidth is-bottomless">
+  <Toggle
+    @name="show-build-crl"
+    @status="success"
+    @size="small"
+    @onChange={{action (mut this.buildCrl)}}
+    @checked={{this.buildCrl}}
+  >
+    {{#if this.buildCrl}}
+      {{#let (get this.config (concat this.section "Attrs")) as |crlAttrs|}}
```

Comment:
This `#let` feels a little wonky but since we're renovating this section of the codebase soon, I went for what felt like the most straightforward solution working with what exists.

The `pki-config` adapter relies on each attribute to be a `field` so it updates correctly. So `disable` has to be included as a key on `crlAttrs`, but instead of rendering a `FormField` for it, we set it by the value of `buildCrl`